pipenv/cli.py

```diff
@@ -407,6 +417,8 @@ def do_create_virtualenv(three=None, python=None):
             python = 'python2'
         elif three is True:
             python = 'python3'
+        else:
+            python = get_python_from_requires()
```

Comment:
We also are going to need testing built around this. It likely won't catch a lot of the edge cases we're concerned about, but it'll be good to have the expected best case behaviour guaranteed.